app/src/main/java/com/example/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -23,6 +25,12 @@ public void showDialogFragment(ListenerDialogFragment listenerDialogFragment) {
         dialogFragment.show(fragmentManager, "dialog_fragment_spaces_setting");
     }
 
+    @Override
+    public void showReservationActivity() {
+        Intent intent = ReservationActivity.newInstance(getContext());
+        getActivity().startActivity(intent);
```

Comment:
please validate that the getContext() and getActivity() is not null before using it

Reply:
done